tests/components/ps4/test_config_flow.py

```diff
@@ -159,26 +168,20 @@ async def test_multiple_flow_implementation(hass):
         "pyps4_2ndscreen.Helper.has_devices",
         return_value=[{"host-ip": MOCK_HOST}, {"host-ip": MOCK_HOST_ADDITIONAL}],
     ):
-        result = await flow.async_step_link(MOCK_CONFIG)
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"], user_input=MOCK_CONFIG
+        )
     assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
     assert result["data"][CONF_TOKEN] == MOCK_CREDS
     assert result["data"]["devices"] == [MOCK_DEVICE]
     assert result["title"] == MOCK_TITLE
 
-    await hass.async_block_till_done()
-
-    # Add entry using result data.
-    mock_data = {
-        CONF_TOKEN: result["data"][CONF_TOKEN],
-        "devices": result["data"]["devices"],
-    }
-    entry = MockConfigEntry(domain=ps4.DOMAIN, data=mock_data)
-    entry.add_to_manager(manager)
-
     # Check if entry exists.
-    assert len(manager.async_entries()) == 1
+    entries = hass.config_entries.async_entries()
+    assert len(entries) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5422434)